doc/classes/PathFollow.xml

```diff
@@ -51,5 +51,8 @@
 		<constant name="ROTATION_XYZ" value="3" enum="RotationMode">
 			Allows the PathFollow to rotate in any axis.
 		</constant>
+		<constant name="ROTATION_ORIENTED" value="4" enum="RotationMode">
```

Comment:
Same here, you can use `[member Curve3D.up_vector_enabled]`.

Reply:
ok